students/20th/YeonjuHa/westagram/login.js

```diff
@@ -0,0 +1,19 @@
+const loginButton = document.getElementsByTagName('button')[0];
+const idInput = document.querySelector('.idInput');
+const pwInput = document.querySelector('.pwInput');
+
+function active(){
+  if(idInput.value && pwInput.value) {
+    loginButton.removeAttribute("disabled","");
+    loginButton.style.backgroundColor = "#5586EB";
+  } else {
+    loginButton.setAttribute("disabled","");
+    loginButton.style.backgroundColor = "#c4e1fb";
+  }
+}
+
+idInput.addEventListener('keyup', active);
+pwInput.addEventListener('keyup', active);
```

Comment:
기왕 form 태그를 사용하고 계시니, form 태그에 접근하여 이벤트 리스너를 할당해 보세요! 그러면 인풋에 일일이 이벤트를 달지 않아도 됩니다.

```suggestion
loginForm.addEventListener('keyup', enableLogin);
```